Marlin/Configuration.h

```diff
@@ -392,8 +392,9 @@ Here are some standard links for getting your machine calibrated:
  // ---------------------------------
  // 0 - No Pressure Sensor Used
  // 1 - Built-in Pressure Transducer (ITV0050-2UL)
+ // 2 - Built-in Pressure Transducer (Regtronics M5)
 
-  #define E_REGULATOR_SENSOR 1
+  #define E_REGULATOR_SENSOR 2
```

Comment:
We might want to consider using defines for 1 and 2 for clarity but with this comment it's not a huge deal